build-api/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM isldocker/build:v5
+
+RUN eval `ssh-agent -s` && \
+    ssh-add ~/.ssh/id_rsa && \
+    ssh-keyscan github.com >> /root/.ssh/known_hosts
+
+RUN git clone -b master git@github.com:istrategylabs/franklin-test.git
+RUN cd franklin-test && git checkout 7b3af09db495427d067704b5e9290d8b81d682c5 
+
+
+RUN cd franklin-test && npm install
+RUN cd franklin-test && npm run build
```

Comment:
Did you mean to commit this? We probably might want to add this to the .gitignore